exasol/toolbox/nox/_artifacts.py

```diff
@@ -118,3 +119,52 @@ def _validate_coverage(path: Path) -> str:
             f"Invalid database, the database is missing the following tables {missing}"
         )
     return ""
+
+
+@nox.session(name="artifacts:copy", python=False)
+def copy_artifacts(session: Session) -> None:
+    """
+    Copy artifacts to the current directory
+    """
+
+    def get_suffix() -> str:
+        versions = getattr(PROJECT_CONFIG, "python_versions", None)
+        pivot = versions[0] if versions else "3.9"
+        return f"-python{pivot}"
+
+    dir = Path(session.posargs[0])
+    suffix = _python_version_suffix()
+    _combine_coverage(session, dir, f"coverage{suffix}*/.coverage")
+    _copy_artifacts(
+        dir,
+        dir.parent,
+        f"lint{suffix}/.lint.txt",
+        f"lint{suffix}/.lint.json",
+        f"security{suffix}/.security.json",
+    )
+
+
+def _python_version_suffix() -> str:
```

Comment:
Using the first version in the python_versions sounds good. Would we maybe want to put that aspect of the function in a more general location? 

Asking as we have a few workflow steps that use a default Python, like 3.10, which may not be compatible in some of the AI projects as you move to newer Python versions. A non-exhaustive list of ones using a default Python version from the python-environment action are: Version-Check, Documentation, Changelog, & Format.

Reply:
Hm, this is a bit trickier given the toolbox's current structure. In such cases, we could typically define a property in the Config class, but as it stands, each repo defines it's own, similar yet differing Config class. We could change that & switch more clearly define what attributes (or switch to properties & have abstract ones) are needed for what & which ones are always required. But that would be outside the scope of this.

For now, we could define the min python version in the `_shared.py`

Reply:
I moved the min python version to file `_share.py` for now.

For all the other topics, I am skeptical about coming up with a final solution for all times.

Maybe an incremental, iterative approach is more efficient? 
I.e. change now only what we see immediately and have a clear proposal for mitigation and adapt as soon as we observe problems or identify additional requirements.